xmodule/library_content_block.py

```diff
@@ -118,20 +123,20 @@ def source_library_key(self):
         return LibraryLocator.from_string(self.source_library_id)
 
     @property
-    def is_source_lib_migrated_to_v2(self):
+    def is_source_lib_migrated_to_v2(self) -> bool:
         """
         Determines whether the source library has been migrated to v2.
         """
-        from cms.djangoapps.modulestore_migrator.api import is_successfully_migrated
+        from cms.djangoapps.modulestore_migrator.api import is_forwarded
 
         return (
             self.source_library_id
             and self.source_library_version
-            and is_successfully_migrated(self.source_library_key, source_version=self.source_library_version)
+            and is_forwarded(self.source_library_key)
```

Comment:
@bradenmacdonald sorry, I lost track of your comment about keeping an `is_successfully_migrated` API function in order to make the API more obvious. i think that's a good idea.

I'm trying to keep "forwarded" and "migrated" distinct (the former being the authoritative thing), so I've gone with `is_forwarded`.

I'm planning to merge later tonight. Let me know if you'd like to see anything different (or I can follow up with a fix on master).